packages/server/modules/core/tests/integration/users.graph.spec.ts

```diff
@@ -199,11 +199,11 @@ describe('Users @graphql', () => {
       ])
     })
 
-    it('doesnt work with less than 3 characters', async () => {
+    it('doesnt work with less than 1 character', async () => {
       const res = await search({
         query: 'fi'
```

Comment:
this isnt right, the query should be invalid, so less than 1 character

Reply:
Ah yes, adjusted